GNOME #410194; Debian #445613; LP #128232; fix crash when displaying tooltips
in SWT apps such as Eclipse

--- gtk+2.0-2.12.0.orig/gtk/gtktooltips.c	2007-09-13 22:35:41.000000000 +0200
+++ gtk+2.0-2.12.0/gtk/gtktooltips.c	2007-10-14 19:47:54.000000000 +0200
@@ -230,6 +230,7 @@
     }
   
   if (tooltips->active_tips_data 
+      && tooltipsdata
       && tooltips->active_tips_data->widget == widget
       && GTK_WIDGET_DRAWABLE (tooltips->active_tips_data->widget))
     {
